openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/BallerinaAuthConfigGenerator.java

```diff
@@ -208,6 +212,7 @@ public class BallerinaAuthConfigGenerator {
     private String clientCredGrantTokenUrl;
     private String passwordGrantTokenUrl;
     private String refreshTokenUrl;
+    private String httpVersion = "http:HTTP_2_0";
```

Comment:
I prefer making this a constant (i.e. `private static final string DEFAULT_HTTP_VERSION = 2.0` and, retrieving the value from the `httpVersionMap` using that key, in here)

Reply:
This variable cannot be modified as a final variable since the value can be changed if user has given a httpVersion in `x-ballerina-http-configurations` extension. The assigned value is updated to get the value from `httpVersionMap`

Reply:
@aneeshafedo yes but we can still define a constant as `private static final string DEFAULT_HTTP_VERSION = "2.0";`
and, use it as `private String httpVersion = httpVersionMap.get(DEFAULT_HTTP_VERSION);`.

Reply:
Sorry for the confusion earlier. Added the suggestion with a small change. The `DEFAULT_HTTP_VERSION` constant is added to the `GeneratorConstants.java` class not to the `BallerinaAuthConfigGenerator.java` since we maintain all the constants in the `GeneratorConstants.java` class.

Reply:
that's exactly what I wanted :)